WindowsServerDocs/remote/remote-access/directaccess/DirectAccess-Known-Issues.md

```diff
@@ -10,8 +10,19 @@ ms.date: 08/07/2020
 ---
 # DirectAccess Known Issues
 
->Applies To: Windows Server (Semi-Annual Channel), Windows Server 2016
+>Applies To: Windows Server (Semi-Annual Channel), Windows Server 2016, Windows Server 2019
 
+## DNS registration of DirectAccess client IPv6 addresses
+
+Starting with the Windows 10 May 2020 Update, a client no longer registers its IP addresses on DNS servers configured in a Name Resolution Policy Table (NRPT).
+If DNS registration is needed, for example **Manage Out**, it can be explicitly enabled with this registry key on the client:
+
+Path: `HKLM\System\CurrentControlSet\Services\Dnscache\Parameters`
+Type: `DWORD`
+Value name: `DisableNRPTForAdapterRegistration`
+Values:
+`1` - DNS Registration disabled (default since the Windows 10 May 2020 Update)
+`0` - DNS Registration enabled
```

Comment:
Like in your other PRs #5207 and #5208, we need to add HTML line breaks (`<br/>`) to keep the line breaks within the Note blob:
```suggestion
Path: `HKLM\System\CurrentControlSet\Services\Dnscache\Parameters`<br/>
Type: `DWORD`<br/>
Value name: `DisableNRPTForAdapterRegistration`<br/>
Values:<br/>
`1` - DNS Registration disabled (default since the Windows 10 May 2020 Update)<br/>
`0` - DNS Registration enabled
```
Feel free to comment or ask questions, in case there is anything in this PR you want to discuss.